### PR TITLE
Added a "Presentations material" section on the web site

### PR DESCRIPTION
**Pull request by ppatierno**

### CODE REVIEW

File: site/content/presentations-material.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
++++
+title = "Presentations Material"
+menu = "main"
+weight = 620
```
Comment (sophokles73) [inline site/content/presentations-material.md]: we already have a page at this position in the menu (`contributing.md`).
I would suggest to position this page between `Architecture` and `Developing Hono` which would result in a weight of  e.g. 590

File: site/content/presentations-material.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
++++
+
+In this page you can find some links to presentations material about Eclipse Hono showed during events, conferences and meetups.
```
Comment (sophokles73) [inline site/content/presentations-material.md]: what about
`On this page you can find links to arbitrary material covering Eclipse Hono which has been presented at events, conferences and meetups.`

File: site/content/presentations-material.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## "Eclipse Hono: Connect. Command. Control." at Virtual IoT Meetup
+
+[Virtual IoT Meetup](https://www.meetup.com/Virtual-IoT/events/242190258/) hosted by Eclipse Foundation (October 11, 2017)
+
+* Speakers : Kai Hudalla (Chief Software Architect at BoschSI), [Paolo Patierno](https://twitter.com/ppatierno) (Senior Software Engineer at Red Hat)
```
Comment (sophokles73) [inline site/content/presentations-material.md]: please make it `Bosch Software Innovations` and (at least for me) leave out the *title*.

File: site/content/presentations-material.md (hunk 1 above)
Comment (sophokles73) [reply site/content/presentations-material.md]: I think we should also rename it to something like  `Presentations et al.`

Comment (ppatierno) [reply site/content/presentations-material.md]: What's the way for defining the position ? I'm taking a look at `config.toml` (is it the right file ?) and I see `Architecture` but not `Developing Hono` :(

Comment (sophokles73) [reply site/content/presentations-material.md]: You can either have explicit entries in the menu in order to support multiple pages under that entry (like we do for Architecture and Deplyoment etc)  or you can define a single page and use the front matter to define it as a (single page) entry in menu (like we do here). The position is then determined by the weight indicated in the front matter.

Comment (ppatierno) [reply site/content/presentations-material.md]: Ok ! I didn't understood that `weight` was the right parameter for that :-) 
Regarding the title maybe just `Presentations` or something different ? Using Latin sounds not to good to me :-)

**Pull request merged**